csrc/scheduler/matmul.cpp

```diff
@@ -1060,11 +1105,17 @@ void scheduleMatmul(Fusion* fusion, const MatmulParams& params) {
     scheduleFusionInputsForEpilogue(roles_map, params.use_smem_epilogue);
   }
 
+  if (num_splitk_dims) {
+    // Inline the splitk sum with the output store
+    splitk_sum->computeAt(d, -2);
```

Comment:
`computeAt` is deprecated, not code should not use it.

Reply:
What should I replace it with here? I tried various versions of `inlineAt` and `inlineSelectedAt`. I couldn't get it to inline with the output store, so I just went with this for the time being but you're right it should be updated.

Reply:
Would do a propagation before inlining help?

Reply:
I have tried to get a propagation to work but it's tricky to mimic computeAt. The proper solution is probably to just figure out why `inlineMost` is not inlining `splitk_sum` with `d` and then address that. I'll keep trying.